cpp/src/groupby/hash/groupby.cu

```diff
@@ -96,11 +96,11 @@ bool constexpr is_hash_aggregation(aggregation::Kind t)
   // this is a temporary fix due to compiler bug and we can resort back to
   // constexpr once cuda 10.2 becomes RAPIDS's minimum compiler version
   // return array_contains(hash_aggregations, t);
-  return (t == aggregation::SUM) or (t == aggregation::MIN) or (t == aggregation::MAX) or
-         (t == aggregation::COUNT_VALID) or (t == aggregation::COUNT_ALL) or
-         (t == aggregation::ARGMIN) or (t == aggregation::ARGMAX) or
-         (t == aggregation::SUM_OF_SQUARES) or (t == aggregation::MEAN) or
-         (t == aggregation::STD) or (t == aggregation::VARIANCE);
+  return (t == aggregation::SUM) or (t == aggregation::PRODUCT) or (t == aggregation::MIN) or
+         (t == aggregation::MAX) or (t == aggregation::COUNT_VALID) or
+         (t == aggregation::COUNT_ALL) or (t == aggregation::ARGMIN) or
+         (t == aggregation::ARGMAX) or (t == aggregation::SUM_OF_SQUARES) or
+         (t == aggregation::MEAN) or (t == aggregation::STD) or (t == aggregation::VARIANCE);
```

Comment:
Can you order the ENUM alphabetically, please? Like
```
(t == aggregation::ARGMAX) or \
(t == aggregation::COUNT_VALID) or \
..
(t == aggregation::MEAN) or \
(t == aggregation::VARIANCE);
```

Reply:
I think that like aggregations should be grouped, not alphabetical. Alphabetical order is not relevant here.

Reply:
e.g.
SUM,
PRODUCT,
SUM_OF_SQUARES,
ARGMIN,
ARGMAX,
MIN,
MAX,
COUNT_ALL,
COUNT_VALID,
MEAN,
STD,
VARIANCE

Reply:
They can be grouped and sorted, so when you want to check if something is there, you can do binary search with your eye. Otherwise, you have to do linear scan the whole list.